lib/active_force/association/eager_load_projection_builder.rb

```diff
@@ -54,8 +55,8 @@ def apply_association_scope(query)
       # to be pluralized
       def query_with_association_fields
         relationship_name = association.sfdc_association_field
-        query = ActiveQuery.new(association.relation_model, relationship_name)
-        query.fields association.relation_model.fields
```

Comment:
`query.fields association.relation_model.fields` is legacy when this used to be a query, not an ActiveQuery. It doesn't actually do anything.